File: backend/KanriSocial/src/SocialMediaManager.Shared/Dtos/Facebook/FacebookPageData.cs
namespace SocialMediaManager.Shared.Dtos.Facebook;

public record FacebookPageData(
    string Id,
    string About,
    string Category,
    List<Category> CategoryList,
    int Checkins,
    ConnectedInstagramAccount ConnectedInstagramAccount,
    int FollowersCount,
    bool IsPublished,
    string Link,
    string Name,
    int RatingCount
);

File: frontend/src/types/facebook/facebook-page-data.ts
import { Category } from "@/types/facebook/facebook-user-page";

export interface ConnectedInstagramAccount {
    id: string;
}

export interface FacebookPageData {
    id: string;
    about: string;
    category: string;
    categoryList: Category[];
    checkins: number;
    connectedInstagramAccount: ConnectedInstagramAccount;
    followersCount: number;
    isPublished: boolean;
    link: string;
    name: string;
    ratingCount: number;
}